linked-events.swagger.yaml

```diff
@@ -1,1749 +1,2912 @@
+        created_by:
+          type: string
+          description: Name and email of the user who created this event. Only available for authorized users in the publisher organization hierarchy.
+          readOnly: true
+        last_modified_by:
+          type: string
+          description: Name and email of the user who last edited this event. Only available for authorized users in the publisher organization hierarchy.
+          readOnly: true
+        date_published:
+          type: string
+          description: Date this event is free to be published
+          format: date-time
+        start_time:
+          type: string
+          description: Time the event will start
+          format: date-time
+        end_time:
+          type: string
+          description: Time the event will end
+          format: date-time
+        custom_data:
+          type: string
+          description: |
+            Key value field for custom data.
+        user_name:
+          type: string
+          description: |
+            Name of the external user.
+        user_email:
+          type: string
+          description: |
+            Email of the external user.
+        user_phone_number:
+          type: string
+          description: |
+            Phone number of the external user.
+        user_organization:
+          type: string
+          description: |
+            Organization of the external user.
+        user_consent:
+          type: boolean
+          description: |
+            Consent to user information of the external user.
+        environment:
+          type: string
+          description: Environment of the event. Current options are in (Indoor) and out (Outdoor).
+          enum:
+            - in
+            - out
+        environmental_certificate:
+          type: string
+          description: |
+            Url of the environmental certificate.
+        audience_min_age:
+          type: integer
+          description: Minimum age of attendee
+          example: 8
+        audience_max_age:
+          type: integer
+          description: Maximum age of attendee
+          example: 18
+        super_event_type:
+          type: string
+          description:
+            If the event has sub_events, describes the type of the event.
+            Current options are 'null', 'recurring', which means a repeating event, and 'umbrella', which means a major event that has subevents.
+          enum:
+            - recurring
+            - umbrella
+        deleted:
+          type: boolean
+          description: Whether this event has been deleted in the original data source
+        maximum_attendee_capacity:
+          type: integer
+          description: Maximum number of people allowed to enrol for the course
+          example: 100
+        minimum_attendee_capacity:
+          type: integer
+          description: Minimum number of people required to enrol for the course
+          example: 10
```

Comment:
https://github.com/City-of-Helsinki/linkedevents/commit/3e5148314009110510750840c7d22c25b12a2705

added

maximum_attendee_capacity_gte
minimum_attendee_capacity_gte
maximum_attendee_capacity_lte
minimum_attendee_capacity_lte


Also is this field usage restricted to the type "course" or should these read just generically "event"?

Reply:
Added instructions to filter by attendee capacity in https://github.com/City-of-Helsinki/api-linked-events/pull/13/commits/1621362a824ca7c2adaf1acf0b6d37ac42518dc0